app/views/sandbox/budget_guide.html.erb

```diff
@@ -0,0 +1,269 @@
+<div class="column">
+
+  <main class="content">
+    <header class="gradient-bg">
+      <div class="inner">
+        <h1>Así funciona un presupuesto municipal</h1>
+        <p>El presupuesto municipal es el documento que explica cómo se gestiona el dinero público durante un año. Aquí te explicamos los conceptos más importantes para que no te pierdas en su lectura.</p>
+
+        <ul>
+          <li><a href="#como-se-lee">¿Cómo se lee un presupuesto?</a></li>
+          <li><a href="#tipos-de-ingreso">Tipos de ingreso</a></li>
+          <li><a href="#tipos-de-gasto">Tipos de gasto</a></li>
+        </ul>
+      </div>
+    </header>
+
+    <article>
+      <%= image_tag 'illustrations/presupuestos.jpg', class: 'pure-img' %>
```

Comment:
Right now the image is being pixalated. For this specific one, I would not make it so wide.